src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 humanYear when animals age 0', () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test('should return 0 humanYear when animals age <= 14', () => {
```

Comment:
The test description says 'should return 0 humanYear when animals age <= 14', but the test checks for age 14 only. You should add more test cases to cover the range of ages up to and including 14.